src/Controllers/ElementalAreaController.php

```diff
@@ -138,7 +138,9 @@ public function save(array $data, Form $form): HTTPResponse
         // Remove the namespace prefixes that were added by EditFormFactory
         $dataWithoutNamespaces = static::removeNamespacesFromFields($data, $element->ID);
 
-        // Update and write the data object which will trigger model validation
+        // Update and write the data object which will trigger model validation.
+        // Would usually be handled by $form->saveInto($element) but since the field names
+        // in the form have been namespaced, we need to handle it ourselves.
         $element->updateFromFormData($dataWithoutNamespaces);
```

Comment:
Really we should just rename the fields _in the form_, and then call `$form->saveInto($element)` - but doing that now would be an API break because someone may have customised the `updateFromFormData()` method in their custom elements.